docs/source/endpoints/inherit.md

```diff
@@ -0,0 +1,57 @@
+---
+myst:
+  html_meta:
+    "description": "The inherit endpoint shows values inherited from content items higher in the hierarchy."
+    "property=og:description": "The inherit endpoint shows values inherited from content items higher in the hierarchy."
+    "property=og:title": "Inherit behaviors"
+    "keywords": "Plone, plone.restapi, REST, API, inherit, acquisition, behaviors"
+---
+
+(inherit-behaviors-label)=
+
+# Inherit behaviors
+
+Plone content is arranged in a hierarchy: a content item has a parent, which has its own parent, all the way up to the Plone site root.
+Together, all these parents are _ancestors_.
```

Comment:
```suggestion
Plone content items are arranged in a hierarchy.
Each content item has a parent, each of which may have its own parent, and continue all the way to the Plone site root.
Together, the chain of parents are _ancestors_ of the content item.
```

Reply:
@stevepiercy Sorry, I'm in a bit of a hurry today. Your edits are good. Committed directly to `main` in https://github.com/plone/plone.restapi/commit/8816d589c3fae1fbbb48378a345b179e4f38e15c